sparktk-core/src/test/scala/org/trustedanalytics/sparktk/models/timeseries/max/MaxModelTest.scala

```diff
@@ -0,0 +1,145 @@
+/**
+ *  Copyright (c) 2016 Intel Corporation 
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.trustedanalytics.sparktk.models.timeseries.max
+
+import org.apache.spark.sql.Row
+import org.apache.spark.sql.catalyst.expressions.GenericRow
+import org.scalatest.Matchers
+import org.trustedanalytics.sparktk.frame.{ Column, DataTypes, Frame, FrameSchema }
+import org.trustedanalytics.sparktk.models.timeseries.max.MaxModel
+import org.trustedanalytics.sparktk.testutils.TestingSparkContextWordSpec
+
+class MaxModelTest extends TestingSparkContextWordSpec with Matchers {
+  val rows: Array[Row] = Array(
+    new GenericRow(Array[Any](68, 278, 0, 28, 0.015132758079119)),
```

Comment:
Is this the data from Kyle?  If so, we can't use it in the test - use Air Quality or something else instead